src/templates/artist-page.jsx

```diff
@@ -98,17 +105,17 @@ const ArtistPageTemplate = ({ data, location }) => {
             </h3>
             <Video url={videoTag} />
           </section>
-          <section className="artist__map">
+          <section className="gallery">
             <h3>
-              <FormattedMessage id="mapTitle" />
+              <FormattedMessage id="galleryTitle" />
             </h3>
-            <Geowidget url={geoTag} />
+            <Gallery images={galleryImages} />
           </section>
-          <section className="gallery">
+          <section className="artist__map">
```

Comment:
Such class name seems strange to me. Has it been here all this time and I haven't noticed?